### PR TITLE
[SLA] PIM-5638: fix wrong category display on published product grid

### PR DESCRIPTION
**Pull request by solivier**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | No
| Added Behats                      | No
| Changelog updated                 | No
| Review and 2 GTM                  | No

### CODE REVIEW

**Comment by jjanvier** [comment]
gtm

File: src/Pim/Bundle/DataGridBundle/Resources/views/macros.html.twig [hunk 1]
```diff
@@ -42,8 +42,8 @@
                         urlParams['{{ name }}[_parameters][view][columns]'] = state.columns;
                     }
 
-                    {% if name == 'product-grid' %}
-                        var categoryFilter = new CategoryFilter(urlParams, '{{ name }}');
+                    {% if categoryBaseRoute is not null %}
+                                        var categoryFilter = new CategoryFilter(urlParams, '{{ name }}', '{{ categoryBaseRoute }}');
```
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Resources/views/macros.html.twig]: extra extra extra spaces :)

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/tree-view.jstree.js [hunk 2]
```diff
@@ -213,6 +214,7 @@ define(
                 selectedNode  = _.has(state, 'selectedNode') ? state.selectedNode : selectedNode;
                 selectedTree  = _.has(state, 'selectedTree') ? state.selectedTree : selectedTree;
                 includeSub    = _.has(state, 'includeSub')   ? state.includeSub   : includeSub;
+                categoryBaseRoute = baseRoute;
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/tree-view.jstree.js]: baseRoute is useless

File: src/Pim/Bundle/DataGridBundle/Resources/views/macros.html.twig (hunk 1 above)
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Resources/views/macros.html.twig]: add categoryBaseRoute seems useless

**Comment by skeleton** [comment]
gtm once fixed

**Pull request merged**